feature/experimental/interface/my_station_mac/ate_tests/my_station_mac_test/my_station_mac_test.go

```diff
@@ -0,0 +1,283 @@
+		IPv6:    "2001:db8::1",
+		IPv4Len: ipv4PrefixLen,
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	dutSrc = attrs.Attributes{
+		Desc:    "DUT to ATE source",
+		IPv4:    "192.0.2.2",
+		IPv6:    "2001:db8::2",
+		IPv4Len: ipv4PrefixLen,
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	dutDst = attrs.Attributes{
+		Desc:    "DUT to ATE destination",
+		IPv4:    "192.0.2.5",
+		IPv6:    "2001:db8::5",
+		IPv4Len: ipv4PrefixLen,
+		IPv6Len: ipv6PrefixLen,
+	}
+
+	ateDst = attrs.Attributes{
+		Name:    "ateDst",
+		IPv4:    "192.0.2.6",
+		IPv6:    "2001:db8::6",
+		IPv4Len: ipv4PrefixLen,
+		IPv6Len: ipv6PrefixLen,
+	}
+)
+
+// configInterfaceDUT configures the DUT interfaces.
+func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes) *oc.Interface {
+	i.Description = ygot.String(a.Desc)
+	i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	if *deviations.InterfaceEnabled {
+		i.Enabled = ygot.Bool(true)
+	}
+
+	s := i.GetOrCreateSubinterface(0)
+	s4 := s.GetOrCreateIpv4()
+	if *deviations.InterfaceEnabled && !*deviations.IPv4MissingEnabled {
+		s4.Enabled = ygot.Bool(true)
+	}
+	s4a := s4.GetOrCreateAddress(a.IPv4)
+	s4a.PrefixLength = ygot.Uint8(ipv4PrefixLen)
+
+	s6 := s.GetOrCreateIpv6()
+	if *deviations.InterfaceEnabled {
+		s6.Enabled = ygot.Bool(true)
+	}
+	s6.GetOrCreateAddress(a.IPv6).PrefixLength = ygot.Uint8(ipv6PrefixLen)
+
+	return i
+}
+
+// configureDUT configures port1 and port2 on the DUT.
+func configureDUT(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	d := gnmi.OC()
+
+	p1 := dut.Port(t, "port1")
+	i1 := &oc.Interface{Name: ygot.String(p1.Name())}
+	gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), configInterfaceDUT(i1, &dutSrc))
+
+	p2 := dut.Port(t, "port2")
+	i2 := &oc.Interface{Name: ygot.String(p2.Name())}
+	gnmi.Replace(t, dut, d.Interface(p2.Name()).Config(), configInterfaceDUT(i2, &dutDst))
+
+	if *deviations.ExplicitPortSpeed {
+		fptest.SetPortSpeed(t, p1)
+		fptest.SetPortSpeed(t, p2)
+	}
+	if *deviations.ExplicitInterfaceInDefaultVRF {
+		fptest.AssignToNetworkInstance(t, dut, p1.Name(), *deviations.DefaultNetworkInstance, 0)
+		fptest.AssignToNetworkInstance(t, dut, p2.Name(), *deviations.DefaultNetworkInstance, 0)
+	}
+}
+
+// configureATE configures port1 and port2 on the ATE.
+func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
+	top := ate.Topology().New()
+
+	p1 := ate.Port(t, "port1")
+	i1 := top.AddInterface(ateSrc.Name).WithPort(p1)
+	i1.IPv4().
+		WithAddress(ateSrc.IPv4CIDR()).
+		WithDefaultGateway(dutSrc.IPv4)
+	i1.IPv6().
+		WithAddress(ateSrc.IPv6CIDR()).
+		WithDefaultGateway(dutSrc.IPv6)
+
+	p2 := ate.Port(t, "port2")
+	i2 := top.AddInterface(ateDst.Name).WithPort(p2)
+	i2.IPv4().
+		WithAddress(ateDst.IPv4CIDR()).
+		WithDefaultGateway(dutDst.IPv4)
+	i2.IPv6().
+		WithAddress(ateDst.IPv6CIDR()).
+		WithDefaultGateway(dutDst.IPv6)
+
+	return top
+}
+
+// addRoute adds an IPv4Entry and verifies the same through AFT telemetry.
+func addRoute(t *testing.T, clientA *gribi.Client) {
```

Comment:
The test plan does not say use gRIBI, and this test is part of the interface feature not gRIBI. Can we just configure the static route in network instance?

Reply:
By the way, if ATE just sends traffic from ATE port1's IP to port2's IP, then we don't even need an explicit route. This is what the static ARP test does.

https://github.com/openconfig/featureprofiles/blob/main/feature/interface/staticarp/ate_tests/static_arp_test/static_arp_test.go

Reply:
gRIBI is used because of below:

## Protocol/RPC Parameter Coverage
*  For gRIBI use the same parameters as in TE-2.1.

Reply:
@xw-g I see "For gRIBI use the same parameters as in TE-2.1" in the original version of TE-1.2, but I thought your proposed update to TE-1.2 changed the "Protocol/RPC Parameter Coverage" to N/A. Did you leave this in by mistake? I thought we had agreed that TE-1.2 can be tested without gRIBI.

Reply:
I think it's left there by mistake. The test body does not mention any about gRIBI. b/268350377 was opened last week with Juniper team to correct it. Will follow up there.

Reply:
@xw-g: Seems b/268350377 still mentions about installing static route. My understanding per Likai's comment ( https://github.com/openconfig/featureprofiles/pull/1020#discussion_r1102108125) is that we don't even need any explicit/static route. Please clarify. Thanks.

Reply:
I'm ok with removing the static route part (because the focus of the test is about My Station Mac, which is about the ingress)

Reply:
Thanks @xw-g for the clarification. I will make the changes accordingly.

Reply:
Changes committed. Please review. Thanks.